internal/controller/suite_test.go

```diff
@@ -66,6 +67,11 @@ var _ = BeforeSuite(func() {
 			fmt.Sprintf("1.29.0-%s-%s", runtime.GOOS, runtime.GOARCH)),
 	}
 
+	_, set := os.LookupEnv("OPERATOR_NAMESPACE")
+	if !set {
+		Expect(os.Setenv("OPERATOR_NAMESPACE", "ceph-csi-operator-system")).To(Succeed())
```

Comment:
in the AfterSuit lets unset this env if we are setting it

Reply:
Done